rdkafka-sys/src/types.rs

```diff
@@ -246,6 +246,9 @@ pub enum RDKafkaErrorCode {
     AutoOffsetReset = -140,
     /// Partition log truncation detected
     LogTruncation = -139,
+    /// A different record in the batch was invalid and this message failed
+    /// persisting.
+    InvaliidDifferentRecord = -138,
```

Comment:
I think there's typo (two `ii`) in `InvaliidDifferentRecord` :)

Reply:
@hgdeoro Thank you! Fixed.